gulpfile.js

```diff
@@ -77,150 +74,143 @@ gulp.task('serve', function () {
  * More about Nunjucks here:
  * https://mozilla.github.io/nunjucks/
  */
-gulp.task('eleventy', function (cb) {
-  cp.exec('npx eleventy', function (err, stdout, stderr) {
+function eleventy(callback) {
```

Comment:
We talked about this one a bit offline, but the eleventy watch task isn't working anymore. So when you run `npm run start` the local server starts up, but if you make any changes to `index.md` or any of the `.njk` files it doesn't trigger a rebuild of the static HTML.

I recently rewrote the datepicker gulp tasks which where originally based off of this boilerplate. Maybe have a look at how I set up the Eleventy tasks there as a place to start.

https://github.com/indiana-university/rivet-datepicker/blob/master/gulpfile.js